.github/workflows/ci.yml

```diff
@@ -6,9 +6,35 @@ on:
 
 jobs:
   test:
+    timeout-minutes: 10
     runs-on: ubuntu-latest
+
     steps:
-      - uses: DerYeger/pnpm-setup-action@master
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: "v18"
+          cache: "pnpm"
+          cache-dependency-path: "**/pnpm-lock.yaml"
+
+      - uses: pnpm/action-setup@v2
+        name: Install pnpm
+        with:
+          version: 8.2.0
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Install Playwright Browsers
+        run: pnpm exec playwright install --with-deps
```

Comment:
Can we run playwright in a different workflow?

Our motivation is to keep same config files as much as possible along with jotaijs/*.

Reply:
We can, surely.

Reply:
Makes no sense. Look at the latest comment.